green_calibration.py

```diff
@@ -47,13 +47,8 @@ def get_new_hsv(self, hsv, contours):
         s = self.true_green_vals[:, :, 1]
         v = self.true_green_vals[:, :, 2]
 
-        low_h, low_s, low_v = (h.mean() - self.H_STD_TOLERANCE * h.std()),\
-                              (s.mean() - self.S_STD_TOLERANCE * s.std()),\
-                              (v.mean() - self.V_STD_TOLERANCE * v.std())
-
-        high_h, high_s, high_v = (h.mean() + self.H_STD_TOLERANCE * h.std()),\
-                                 (s.mean() + self.S_STD_TOLERANCE * s.std()),\
-                                 (v.mean() + self.V_STD_TOLERANCE * v.std())
+        low_h, low_s, low_v = (h.mean() - self.H_STD_TOLERANCE * h.std()), (s.mean() - self.S_STD_TOLERANCE * s.std()), (v.mean() - self.V_STD_TOLERANCE * v.std())
```

Comment:
I meant let's not use tuples at all, I think it would be much more readable if we do:

```
low_h =...
low_s = ...
low_v = ...
```